odoo/tools/translate.py

```diff
@@ -152,7 +152,7 @@ def encode(s):
     'value_label',
 }, lambda e: True)
 TRANSLATED_ATTRS.update(
-    value=lambda e: (e.tag == 'input' and e.attrib.get('type', 'text') == 'text') and 'datetimepicker-input' not in e.attrib.get('class').split(' '),
+    value=lambda e: (e.tag == 'input' and e.attrib.get('type', 'text') == 'text') and 'datetimepicker-input' not in e.attrib.get('class', '').split(' '),
```

Comment:
The `split(' ')` can probably be omitted but this is just about fixing the forward port